sorts/bucket_sort.py

```diff
@@ -30,7 +30,7 @@
 from __future__ import annotations
 
 
-def bucket_sort(my_list: list) -> list:
+def bucket_sort(my_list: list, bucket_count: int = 10) -> list:
```

Comment:
why bucket count is set to 10?

Reply:
Just some default value. It can be 100 or whatever.  
Or we can set it to `None` by default and calculate it by some heuristic if it's not set, ex. `len(my_list) // 2`.

Reply:
I suggest that rather this having bucket_count as a parameter, we can have it as part of implementation logic where we calculate bucket count based on type of machine 32 for 32bit, 64 for 64bit. In that way we eliminate user driven errors